templates/config.json

```
@@ -441,7 +441,7 @@
       "name": "Assignment of Roles and the Security Committee",
       "type": "administrative",
       "provider": "JupiterOne",
-      "summary": "It is required by regulatory compliance, such as HIPAA, to formally assign security and privacy responsibilities to someone in your organization. As your organization expands and matures, a security committee may be formed. Having an executive level security committee is often a requirement for security certifications such as HITRUST CSF. You can manage the role assignment manually (i.e. in this documented policy/control procedure), or manage roe assignments to users/people in JupiterOne. In most case, you do not need to implement a complicated enterprise GRC system.",
+      "summary": "It is required by regulatory compliance, such as HIPAA, to formally assign security and privacy responsibilities to someone in your organization. As your organization expands and matures, a security committee may be formed. Having an executive level security committee is often a requirement for security certifications such as HITRUST CSF. You can manage the role assignment manually (i.e. in this documented policy/control procedure), or manage role assignments to users/people in JupiterOne. In most cases, you do not need to implement a complicated enterprise GRC system.",
```

```suggestion
      "summary": "Regulatory compliance frameworks, such as HIPAA, require formal assignment of security and privacy responsibilities to someone in your organization. As your organization expands and matures, you may want to form an executive level security committee to meet this requirement. Often, such a committee is a requirement of security certifications such as HITRUST CSF. You can manage this role assignment manually (I.E. in this documented policy/control procedure) or in the JupiterOne's app. Using JupiterOne, you can almost always avoid implementing a complicated enterprise GRC system.
```

I'm not sure if "manage role assignments to users/people in JupiterOne" actually means what I changed it to ("in the JupiterOne's app"), but hopefully the idea came across.

---

@erkangz Do you find this suggestion useful? If you want, I can do one for the other summaries as well, but I understand if you just want to use them as they are now and avoid that extra work.